### DISCUSSION TITLE
artificial intelligence which handles an operating system

### DISCUSSION THREAD

**Discussion by SparshkPavan** [os-js/OS.js#826] [Ideas]
Help! Please cascade as its an idea can make it happen.

**Comment by saoirse-iontach** [comment]
What do you mean ?
I think an OS must be a small kernel providing core services, only essential supports. Then we can provide extra modules to extends it, and then we can use some machine learning to provide automatic orchestration (in example, finding how to provide a view to an url in some context, which can be file property, file history, links, summary, and others...) (other example, how to agence windows) ; but theses are complexe comportment that is no essential, and are unpredictable.

Generalized IA is more likely intelligence which could have a great place in area of development, but yet an other domain, not OS. And Generalized IA is far to be publicly available...

That's said, I would like with pleasure read your arguments :-)

**Comment by SparshkPavan** [comment]
Perfect!

**Comment by Avomixs-Glitch** [comment]
NICE BRO
